.github/workflows/workflow.yml

```diff
@@ -70,7 +70,7 @@ jobs:
           pip install pipenv
           pip install tox
       - name: Integration tests
-        run: tox -e py3.7 -- -m integration
+        run: tox -e py3.7 -- -m integration -m 'not unstable'
```

Comment:
We now have the mark `unstable` for tests which we do not currently have time to fix and display instability in CI.